x11-libs/intel-hybrid-codec-driver/intel-hybrid-codec-driver-2.0.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MULTILIB_COMPAT=( abi_x86_64 )
+EGIT_COMMIT="edead0c17e2818bc0fee0ea644f85ab81bbe6f7a"
+
+inherit autotools multilib-minimal
+
+DESCRIPTION="Libva support for partially hardware accelerated encode and decode on Haswell and newer"
+HOMEPAGE="https://github.com/intel/intel-hybrid-driver"
+SRC_URI="https://github.com/intel/intel-hybrid-driver/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/intel-hybrid-driver-${EGIT_COMMIT}"
+KEYWORDS="~amd64 ~amd64-linux"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="wayland X"
+
+RDEPEND="
+	>=x11-libs/cmrt-1.0.6
+	>=x11-libs/libdrm-2.4.45
+	>=x11-libs/libva-1.0.16[${MULTILIB_USEDEP}]
+"
+DEPEND="${RDEPEND}"
```

Comment:
I did now performed a test emerge locally. This package uses ```pkgconfig, checking pkg-config is at least version 0.9.0... yes```. You need to add ```virtual/pkgconfig``` into ```BDEPEND```. Otherwise, lgtm.